store/snapshots/manager.go

```diff
@@ -550,3 +550,9 @@ func (m *Manager) snapshot(height int64) {
 		m.logger.Debug("pruned state snapshots", "pruned", pruned)
 	}
 }
+
+// close snapshot database
+func (m *Manager) Close() {
```

Comment:
Lets add a changelog under store/CHANGELOG.md for this.

Reply:
In #17294